mypy.ini

```diff
@@ -0,0 +1,8 @@
+[mypy]
+follow_imports=silent
+
+[mypy-numpy.*]
+ignore_missing_imports=True
+
+[mypy-pandas._libs.*]
```

Comment:
Generally doesn't look like Mypy likes Cython imports. Have this in the config but also noted specifically on imports within the code.

There's probably a better approach to this just haven't found yet

Reply:
According to [the PEP](https://www.python.org/dev/peps/pep-0484/#stub-files), you should be able to use .pyi/stub files to annotate compiled extensions. Not ideal though since they'd have to stay in-sync with the extension